### ISSUE TITLE
Plugin shows error in menu entry on Camunda Modeler > 4.5

### ISSUE THREAD

**Issue by rob2universe** [sharedchains/camunda-modeler-plugin-multidiagram#9]

**Comment by Ichigo85** [comment]
Hello. It seems that there are multiple plugins directory in camunda modeler directory.
Please follow this steps:

- download the latest release on github
- copy the folder camunda-modeler-plugin-multidiagram in camunda-modeler/resources/plugins folder

It should work.
Thank you.

**Comment by iha10512** [comment]
Hello Ichgo85,

i downloaded the camunda-modeler-plugin-multidiagram-master.zip and extracted it to Camunda-modeler/resources/plugins folder and I still have the same issue as rob2universe.
I use the Camunda Modeler 4.7.0.

What am I doing wrong?

**Comment by Ichigo85** [comment]
Hi @iha10512 ,
please download the [release zip](https://github.com/sharedchains/camunda-modeler-plugin-multidiagram/releases/download/v1.0.1/camunda-modeler-plugin-multidiagram-1.0.1.zip) and extract it to the same directory. Let me know if you still have issues.

**Comment by iha10512** [comment]
Hi @Ichigo85, thanks for the fast reply.
I downloaded it and extracted it to the same folder. First it was not recognized by the modeler, because the index.js was two folders deep: "camunda-modeler-plugin-miltidiagram-1.0.1/camunda-modeler-plugin-multidiagram/index.js". so I copied the folder: "camunda-modeler-plugin-multidiagram" one level up and deleted the original folder. Now I could test the functions of the plugin. 

First I tested the Internal link function with the "call activity" to some embedded processes I created and it works! 
Second I tested the Internal link function with the "collapsed Sub process" to some embedded processes I created and nothing happend.
Third I tried the External link function with the "call activity" to some other bpmn-files located in the same folder, but nothing happend when I double-clicked. I guess the plugin not meant to work with external files and collapsed Sub processes without call activity?
